docs/global_overview.md

```diff
@@ -0,0 +1,159 @@
+# Global overview
+
+## Executive summary
```

Comment:
an executive summary should start with the why and a bit more context

Reply:
Done